app/javascript/packages/document-capture/components/hybrid-doc-capture-warning.spec.tsx

```diff
@@ -0,0 +1,92 @@
+import { render } from '@testing-library/react';
+import { t } from '@18f/identity-i18n';
+import HybridDocCaptureWarning from './hybrid-doc-capture-warning';
+import { Provider as ServiceProviderContextProvider } from '../context/service-provider';
+
+const APP_NAME = 'Login.gov';
+const SP_NAME = 'TEST SP';
+
+describe('HybridDocCaptureWarning', () => {
+  before(() => {
+    const config = document.createElement('script');
+    config.id = 'test-config';
+    config.type = 'application/json';
+    config.setAttribute('data-config', '');
+    config.textContent = JSON.stringify({ appName: APP_NAME });
+    document.body.append(config);
+  });
+
+  after(() => {
+    const config = document.getElementById('test-config');
+    if (config) {
+      config.remove();
+    }
+  });
```

Comment:
Two related points: We have automatic DOM cleanup logic that runs after each test case, so...

1. The `after` block technically isn't required
2. You'll want this to be `beforeEach`, since the cleanup happens between each test case, and `before` would only run once before the full test suite, so the element would be removed after the first test case and never re-added.

https://github.com/18F/identity-idp/blob/393294ac6a704e49ca770b31fa3076e60db4c37e/spec/javascripts/support/dom.js#L95-L106

Reply:
Ah right, forgot about the auto cleanup.